### PR TITLE
fix: Fixed Failed to load class "org.slf4j.impl.StaticLoggerBinder".

### PR DESCRIPTION
**Pull request by gabrielpadilh4**
Fixed the below message when starting kerberos server and logger levels that not was working:

SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

### CODE REVIEW

**Comment by gabrielpadilh4** [comment]
Hello @kwart , could you please review and approve ?

Thank you.

File: pom.xml [hunk 1]
```diff
@@ -95,10 +95,15 @@
             <artifactId>jcommander</artifactId>
             <version>1.82</version>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>2.0.7</version>
```
Comment (kwart) [inline pom.xml:101]: Could you make the version a property so we don't repeat it?

**Review by kwart — CHANGES_REQUESTED** [review]
Thank you for the fix. I have just a minor change-request to keep it simpler to maintain.

File: pom.xml (hunk 1 above)
Comment (gabrielpadilh4) [reply pom.xml]: Hello @kwart, done. Thank you.

**Review by kwart — APPROVED** [review]

**Pull request merged**